services/postgres/db-manager.sh

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env bash
+set -u
+
+# Script to invoke the database manager lambda function
+# Usage: ./db-manager.sh create|delete stageName dbSecretArn
+
+action="$1"        # create or delete
+stage_name="$2"    # The stage/environment name for the database
+secret_arn="$3"    # ARN of the secrets manager secret for the main dev DB
+lambda_stage="${4:-dev}"  # Stage where the lambda is deployed, defaults to dev
+function_name="postgres-${lambda_stage}-dbManager"
+lambda_version="\$LATEST"
+cli_read_timeout=240
+
+# Ensure all required parameters are provided
+if [[ -z "$action" || -z "$stage_name" || -z "$secret_arn" ]]; then
+  echo "Usage: $0 create|delete stageName dbSecretArn [lambdaStage]" 1>&2
```

Comment:
Should there be some kind of guard even in this script to avoid deleting higher environments? I get that the way we are calling it right now, it wouldn't be used with non dev environments... more just being cautious

Reply:
Yeah I can add a check that bails if stage_name is main/val/prod. I'll probably want to put that protection in the lambda itself, but will also put it here.